spec/unit/csv_builder_spec.rb

```diff
@@ -277,4 +277,52 @@ def view_context
       end
     end
   end
+
+  context 'csv injection' do
+    let(:dummy_controller) do
+      class DummyController
+        def in_paginated_batches(&block)
+          Post.all.each(&block)
+        end
+
+        def view_context
+          MethodOrProcHelper
+        end
+      end
+      DummyController.new
+    end
+
+    let(:builder) do
+      ActiveAdmin::CSVBuilder.new do
+        column(:id)
+        column(:title)
+      end
+    end
+
+    ['=', '+', '-', '@', "\t", "\r"].each do |char|
+      it "prefix ' when column starts with #{char}" do
+        attack = "#{char}1+2"
+
+        escaped_attack = "'#{attack}"
+        escaped_attack = "\"#{escaped_attack}\"" if char == "\r"
+
+        post = Post.create!(title: attack)
+        receiver = []
+        builder.build dummy_controller, receiver
+        line = receiver.last
+        expect(line).to eq "#{post.id},#{escaped_attack}\n"
+      end
+
+      it "take care of field separator" do
```

Comment:
I wasn't sure if this test is meant to be within the loop because of how it's named and it isn't using the char variable. If it's not meant to use it, then it should be moved outside the char loop and the description updated to "accounts for the field separator".

Reply:
Thanks for this. I forgot to use `char` in the test.